src/bananas.js

```diff
@@ -0,0 +1,180 @@
+define([
+    'vellum/xpath',
+], function(
+    xpath
+) {
+    var OUTSIDE_BANANA = 0,
+        INSIDE_BANANA = 1,
+        DELIMITER = "🍌",
```

Comment:
Ha. yeah. I figured this wouldn't actually go out, but it was more fun to code with than boring ` :)